README.md

```diff
@@ -112,6 +116,44 @@ from djangoevents import subscribe_to                   # from eventsourcing.dom
 from djangoevents import EventSourcedRepository         # from eventsourcing.infrastructure.event_sourced_repo import EventSourcedRepository
 ```
 
+### Documenting event schema
+
+Event schema validation is disabled by default. To enable it for the whole project please add `DJANGOEVENTS_CONFIG` to project's `settings.py`:
+
+```python
+DJANGOEVENTS_CONFIG = {
+    ...
+    'EVENT_SCHEMA_VALIDATION': {
+        'ENABLED': True,
+        'SCHEMA_DIR': 'avro',
+    },
+    ...
+}
+
+```
+
+**It is expected that each service will fully document all events emitted through avro schema definitions**. Read more about [avro format specification](https://avro.apache.org/docs/1.7.7/spec.html).
+
+By default djangoevents assumes event schemas will be placed in `avro` folder located at project's root directory as specifed below:
+
+```bash
+$ tree project
+|- src
+|--- manage.py
+|--- ..
+|-avro
+|--- aggragate_name/
+|----- aggregate_name_test_event1_v1.json
```

Comment:
Yes. I have changed to follow what @marcinkiewiczblazej did for Java. Will send a PR your way in a sec.